lca-cli/seedcreator/seedcreator.go

```diff
@@ -193,6 +194,20 @@ func (s *SeedCreator) handleServices() error {
 	})
 }
 
+func GetSeedAdditionalTrustBundleState(ctx context.Context, client runtimeclient.Client) (*seedclusterinfo.AdditionalTrustBundle, error) {
```

Comment:
just wondering if the `GetSeedAdditionalTrustBundleState` funct could be moved under the utils pkg (in the client_helper.go) as well

Reply:
Can't do that without creating an import loop